docs/version_3.md

```diff
@@ -1,13 +1,40 @@
 # Version 3.0!
 
-We have been working hard the past few months on a number of features and improvements we wanted to combine into a major release. Though we intend that 3.0 to be backwards compatible, we thought it was a good idea to move to a new major version because we are changing a core part of how Style Dictionary works. 
+Version 3.0 is now publicly released! We have been working hard the past few months on a number of features and improvements we wanted to combine into a major release. Though we intend that 3.0 to be backwards compatible, we thought it was a good idea to move to a new major version because we are changing a core part of how Style Dictionary works. 
 
-If you would like to get a pre-release of 3.0, use the `next` tag to install Style Dictionary:
+If you are starting a new project, you can install Style Dictionary and it will give you the latest version:
 
-`npm install --save-dev style-dictionary@next`
+`npm install --save-dev style-dictionary`
+
+If you have an existing project, you can upgrade to 3.0 by updating the version in your `package.json` file to `"style-dictionary": "^3.0.0"` and then run `npm install` or you can use the `latest` tag to update both your `package.json` and `package-lock.json` files:
+
+`npm install --save-dev style-dictionary@latest`
 
 If you find any bugs or issues, please file tickets so we can get things fixed before we release it to the wider community. Thanks!
```

Comment:
> If you find any bugs or issues, please file tickets so we can get things fixed before we release it to the wider community. Thanks!

If you find any bugs or issues, please file tickets so we can get things fixed. We have tested and reviewed 3.0 extensively, but there may be things we missed. Thanks!

Reply:
Done and done!